lib/node_modules/@stdlib/math/stats/anova1/lib/make_anova.js

```diff
@@ -0,0 +1,162 @@
+	var treatment; // Index variable
+	var sumSqError;
+	var meanSumSqTreat; // Mean sum of squares
+	var meanSumSqError;
+	var fScore;
+	var numDf;
+	var denDf;
+	var pVal;
+	var out;
+	var opts;
+
+	// Initialize all vars
+	grandMean = 0;
+	sumSqTotal = 0;
+	ssTreatment = 0;
+
+	opts = copy( defaults );
+
+	// First check pre-conditions
+	if ( !isTypedArrayLike( x ) && !isNumberArray( x )) {
+		throw new TypeError( 'invalid first argument. x must be numeric array. Value: `' + x + '`.');
+	}
+
+	// Validate the options
+	if (arguments.length > 2) {
+		err = validate(opts, arguments[2]);
+		if ( err ) {
+			throw err;
+		}
+	}
+
+	if (x.length <= 1) {
+		throw new RangeError( 'length of x must be greater than or equal to 2.');
+	}
+
+	// Check if factor is an array
+	if ( !isArray( factor )) {
+		throw new TypeError(' factor argument must be an array' );
+	}
+
+	if (factor.length <= 1) {
+		throw new RangeError('Second argument must have at least 2 elements');
+	}
+
+	treats = uniqueElems(factor);
+	if (!treats) {
+		throw new RangeError('Second argument must have at least two unique elements');
+	}
+	if (treats.length <= 1) {
+		throw new RangeError('Second argument must have at least 2 unique elements');
+	}
+
+	// Check if the arrays are the same length
+	if ( x.length !== factor.length ) {
+		throw new RangeError( 'Arguments must be the same length' );
+	}
+
+	means = meanTable(x, factor);
+
+	// Get the total mean
+	for (i = 0; i < x.length; i++) {
+		grandMean += x[i];
+	}
+
+	grandMean /= x.length;
+
+	// Now get total ss
+	for (i = 0; i < x.length; i++) {
+		sq = (x[i] - grandMean) * (x[i] - grandMean);
+		sumSqTotal += sq;
+	}
+
+	/* Now get the treatment SS
+	// Groupmean - grandMean squared times number of obs
+	make an array of means so that I may loop through the indices
+	*/
+	sq = 0;
+	for (treatment in means) {
+		if (treatment in means) {
+			// Already have sq defined
+			sq = (means[treatment].mean - grandMean) *
+				(means[treatment].mean - grandMean);
+			ssTreatment += means[treatment].sampleSize * sq;
+		}
+	}
+
+	// Now for sse
+	// Sse = sst - ssTreatment
+	sumSqError = sumSqTotal - ssTreatment;
+
+	// Now get the mean sum squares and mean errer
+	meanSumSqTreat = ssTreatment / (treats.length - 1);
+
+	meanSumSqError = sumSqError / (x.length - treats.length);
+
+	// Now get the f score
+	fScore = meanSumSqTreat / meanSumSqError;
+
+	// Now get the p-value
+	numDf = treats.length - 1;
+	denDf = x.length - treats.length;
+	pVal = 1 - cdf(fScore, numDf, denDf);
+
+	// Now to make the out object
+	out = {};
+	setReadOnly(out, 'treatment_df', treats.length - 1);
+	setReadOnly(out, 'treatment_sum_squares', ssTreatment);
+	setReadOnly(out, 'treatment_mean_sum_squares', meanSumSqTreat);
+	setReadOnly(out, 'error_df', x.length - treats.length);
+	setReadOnly(out, 'error_sum_squares', sumSqError);
+	setReadOnly(out, 'error_mean_sum_squares', meanSumSqError);
+	setReadOnly(out, 'F', fScore);
+	setReadOnly(out, 'P_Value', pVal);
```

Comment:
Change from `P_Value` to `pValue` to match the other test packages.

Reply:
Resolved in `make_anova.js` and `print.js`